<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateMemWidget</class>
 <widget class="QWidget" name="updateMemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>修改菜单权限</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="spacing">
    <number>20</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <item alignment="Qt::AlignRight">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>角色名:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="roleNameLabel">
         <property name="text">
          <string>角色</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>1</number>
         </property>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>角色暂没有的菜单</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="allMemList"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>70</number>
         </property>
         <property name="bottomMargin">
          <number>70</number>
         </property>
         <item>
          <widget class="QToolButton" name="addToolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="delToolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>1</number>
         </property>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>角色将拥有的菜单</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="ownMemList"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item alignment="Qt::AlignRight">
        <widget class="QPushButton" name="cacelButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignLeft">
        <widget class="QPushButton" name="sureButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
